pkg/asset/installconfig/azure/client.go

```diff
@@ -152,3 +157,29 @@ func (c *Client) getProvidersClient(ctx context.Context) (azres.ProvidersClient,
 	client.Authorizer = c.ssn.Authorizer
 	return client, nil
 }
+
+// GetDiskSkus checks if the diskType exists for the given region.
+func (c *Client) GetDiskSkus(ctx context.Context, region string) ([]azsku.ResourceSku, error) {
+	client := azsku.NewResourceSkusClient(c.ssn.Credentials.SubscriptionID)
+	client.Authorizer = c.ssn.Authorizer
+	ctx, cancel := context.WithTimeout(ctx, 30*time.Second)
+	defer cancel()
+
+	var regionPages []azsku.ResourceSku
+
+	for skuPage, err := client.List(ctx); err != nil && skuPage.NotDone(); err = skuPage.NextWithContext(ctx) {
```

Comment:
It seems like the error is not being handled. If `err != nil` we need to return from `GetDiskSkus`